network/src/p2p/handler.rs

```diff
@@ -80,14 +82,19 @@ const RETRY_SYNC_MAX: Duration = Duration::from_secs(10); // T1
 const RTT: Duration = Duration::from_secs(10); // T2
 const WAIT_SYNC: Duration = Duration::from_secs(30); // T3 >> T1 + RTT
 
+pub trait ManagingPeerdb: Send + Sync {
+    fn insert(&self, transaction: DBTransaction, key: &[u8], value: &[u8]);
```

Comment:
Does `ManagingPeerdb` need `DBTransaction`?
The implementation might use a transaction but it seems implementation detail.
I think there is no need to pass the transaction.

Reply:
Hopefully done.